src/libraries/Microsoft.NETCore.Platforms/src/RID.cs

```diff
@@ -40,20 +47,153 @@ public override string ToString()
             return builder.ToString();
         }
 
-        public bool HasVersion()
+        private enum RIDPart : int
+        {
+            Base = 0,
+            Version,
+            Architcture,
```

Comment:
```suggestion
            Architecture,
```

Reply:
Dangerous suggestion,  need to make the change everywhere :)

Reply:
Yeah I realized afterwards, but for typos like this the suggestion displays the diff better 🙈

Reply:
No worries, but you don't get the commit credit 😆